scripts/999-ping-check

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+ping_checker() {
+	GREEN='\033[32m'
+	RED='\033[31m'
+	NC='\033[0m'
+	HOST="$1"
+	HOST_PING="$(ping -w 2 -c 2 $HOST 2>&1 | grep '% packet' | awk -F'[%]' '{print $1}' | awk -F'[ ]' '{print $NF}')"
+	if [ ! -z "$HOST_PING" ]; then
+		if [ $HOST_PING -lt 25 ]; then
+			echo -e "$HOST......${GREEN}OKAY${NC} "
+		else
+			echo -e "$HOST......${RED}NOT OKAY${NC}"
+		fi
+	else 
+		echo -e "$HOST...... ${RED} NOT FOUND${NC}"
+	fi
+}
+ping_checker mil-sub-gumstix.ad.mil.ufl.edu
+ping_checker mil-com-teledyne-p900.ad.mil.ufl.edu
+btrfs filesystem usage /
```

Comment:
`btrfs fi show` seems to be a better command to run as it's more straight to the point of file system usage